<?xml version="1.0"?>
<launch>

	<arg name="namespace" default="bebop" />
   <arg name="ip" default="192.168.42.1" />
 	<arg name="drone_type" default="bebop2" /> <!-- available drone types: bebop1, bebop2 -->
   <arg name="config_file" default="$(find bebop_driver)/config/LowResColor.yaml" />
   <arg name="camera_info_url" default="package://bebop_driver/data/bebop2_640x368_calib.yaml" />
   <group ns="$(arg namespace)">
     	<node pkg="bebop_driver" name="bebop_driver" type="bebop_driver_node" output="screen">
        	<param name="camera_info_url" value="$(arg camera_info_url)" />
         <param name="bebop_ip" value="$(arg ip)" />
         <rosparam command="load" file="$(arg config_file)" />
     	</node>
     	<include file="$(find bebop_description)/launch/description.launch" />
   </group>

  <node name="vicon" pkg="vicon_bridge" type="vicon_bridge" output="screen">
    <param name="stream_mode" value="ClientPull" type="str" />
    <!--param name="datastream_hostport" value="192.168.0.13:801" type="str" /-->
    <param name="datastream_hostport" value="192.168.10.1:801" type="str" />
    <param name="tf_ref_frame_id" value="/world" type="str" />
  </node> 

</launch>
